docs/christmas/hidden-options.md

```diff
@@ -0,0 +1,29 @@
+# Hidden Configuration Options
+
+There are some hidden configuration options that can resolve some potential issues with Christmas+ on some setups, along with provide additional behavior to the plugin, that may be potentially dangerous.
+
+## Click Cooldown
+Using the `clickCooldown` option can change the time allowed between clicks in the GUI in milliseconds.
+
+ <p class="error"><b>WARNING:</b> This setting is dangerous. This is used to prevent duplication bugs, and is set to 250ms to prevent this. Removing this cooldown can harm changes of duplication. 
```

Comment:
Am I blind or GitHub's diff viewer's gone crazy or none of the opening `<p>` tags you've added (it applies to all of them down below) is closed with `</p>`?

Reply:
This is not just you, Forgot to close them myself. Over-relied on VSC's auto complete. This has since been amended for all files.